README.md

```diff
@@ -110,14 +110,24 @@ git clone https://github.com/open-ce/MY_FEEDSTOCK-feedstock.git
 
 ### Installing Packages
 
-After performing a build, a local conda channel will be created. By default, this will be within a folder called `condabuild` (it can be changed using the `--output_folder` argument). After the build, packages can be installed within a conda environment from this local channel. See conda's [documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/index.html) for more information on conda environments.
+After performing a build, a local conda channel will be created. By default, this will be within a folder called `condabuild` (it can be changed using the `--output_folder` argument). After the build, packages can be installed within a conda environment from this local channel. If the packages are built using build_env.py script, then a conda environment file will also be generated which can be used to generate a conda environment with the built packages installed in it. See conda's [documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/index.html) for more information on conda environments.
 
 The following command will install a package named `PACKAGE` from the local conda channel `condabuild` into the currently active conda environment.
 
 ```bash
 conda install -c ./condabuild PACKAGE
 ```
 
+The following command can be used to create a conda environment using a conda environment file.
+
+```bash
+conda env create -f opence-conda-env-py3.6-cpu-openmpi.yaml
```

Comment:
Do we want to use placeholder for filename here?

Reply:
I think exact file name is also okay, as it is an example. We do that for output_folder directory too just above it.